app/controllers/application_controller.rb

```diff
@@ -1,11 +1,11 @@
-require 'application_responder'
-
 # frozen_string_literal: true
 
+require 'application_responder'
+
 class ApplicationController < ActionController::Base
   include Pundit
   self.responder = ApplicationResponder
-  respond_to :html
+  respond_to :html, :json
```

Comment:
Hmm, I don't know if it's worth having this as a default for everything - most things are just made to respond to HTML - willing to be convinced though

Reply:
This is a quick way of turning lots of parts of the app into an api.

Reply:
Are we turning lots of parts of the app into an api, or just one right now?

Reply:
some. It means i can quickly demo thinkings like `birth_record.json` to show we can read from other software.